modules/packer/custom-image/README.md

```diff
@@ -210,24 +210,25 @@ to the console. For example:
 ==> example.googlecompute.toolkit_image: Startup script, if any, has finished running.
 ```
 
-Using the default value for \[var.scopes\]\[#input_scopes\], the output of
-startup script execution will be stored in Cloud Logging. It can be examined
-using the [Cloud Logging Console][logging-console] or with a
-[gcloud logging read][logging-read-docs] command (substituting `<<PROJECT_ID>>`
-with your project ID):
+### Viewing Startup Script Logs
```

Comment:
The focus of #3150 is improving the debugging experience. I don't think relying on the user to fill in the blanks and browse to the Cloud Console in a timely fashion helps that message. I want to see this become a simple section:

```md
## Debugging startup-script failures

> [!NOTE]
> There can be a delay in the propagation of the logs from the instance to
> Cloud Logging, so it may require waiting a few minutes to see the full logs.

If the Packer image build fails, the module will output a `gcloud` command
that can be used directly to review startup-script execution.
```

I suggest createinga follow-up task that uses a `shell-local` provisioner that prints out only a link to the Cloud Logging for live viewing.  For example:

```
https://console.cloud.google.com/logs/query;query=resource.type%3D%22gce_instance%22%0Aresource.labels.instance_id%3D%22562843389805978984%22?project=hpc-toolkit-dev
```

Reply:
Done